NeuroID/build.gradle

```diff
@@ -85,12 +85,16 @@ dependencies {
     testImplementation "androidx.test:core-ktx:1.4.0"
     testImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     testImplementation 'com.google.truth:truth:0.43'
+    testImplementation 'io.mockk:mockk:1.12.0'
     implementation 'androidx.multidex:multidex:2.0.1'
 
     implementation("androidx.lifecycle:lifecycle-process:2.4.1")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1")
     testImplementation 'io.mockk:mockk:1.12.0'
     reactNativeLibImplementation("com.facebook.react:react-native:0.71.0-rc.0")
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
+    implementation 'com.google.code.gson:gson:2.8.9'
+    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
 }
```

Comment:
I think the okhttp dependency is missing.

Reply:
The OKHttp3 library is included transitively with Retrofit (I think it is in the Retrofit .pom file as a dependency).

Reply:
Looks like we only have to tell user to add Retrofit dependency if they include the SDK manually. Retrofit dependency will include OKHTTP3.